### DISCUSSION TITLE
[@types/google-apps-script] is missing TextButtonStyle.OUTLINED

### DISCUSSION THREAD

**Discussion by ak37165** [DefinitelyTyped/DefinitelyTyped#69795] [Issues with a @types package]
https://developers.google.com/apps-script/reference/card-service/text-button-style specifies FILLED and OUTLINED. The types do not expose OUTLINED but expose an incorrect TEXT enum value - this causes scripts to fail at run time.

[a comment by an automated account is not shown]

**Comment by mtgto** [comment]
Thank you for your report.

Release Notes shows that `TEXT` is renamed to `OUTLINED` in Mar 15, 2024.

We are welcome to you create a pull request to fix it! (Because module owners, as I, can not approve PR by myself)

<img width="1245" alt="image" src="https://github.com/DefinitelyTyped/DefinitelyTyped/assets/1213991/1733d091-2c06-4819-a1a2-a9f066140624">
